Tiber upgrade (#259)
autonity/docs.autonity.org: reference/api/aut/index.md

```
:::

## atnTotalRedistributed

> Returns the total amount of Auton staking rewards distributed since genesis minus treasury fee.

Reviewer: Elsewhere in the docs, in this PR as well, there is usage of auton in lower case but it seems that uppercase is more appropriate and should be used more consistently.

Author: This is fixed in the Concepts section (not many changes).  Commit 4dc0780e19c351728dc824ad20187847234b1f2b.

Author: Further fixes in commits dfb84273d3582fd3d2a26342642bbe14a9882b99 and 2cc1eb9d5214030bd5af2836e3f9d258420a3ca0.

aut token balance-of --ntn 0xd4eddde5d1d0d7129a7f9c35ec55254f43b8e6d4
1000.000000000000000000
```
:::

> To return an ERC2-0 contract token (e.g. Liquid Newton) balance for an account specify the `--token` option:

Reviewer 2: ```suggestion
To return an ERC-20 contract token (e.g. Liquid Newton) balance for an account specify the `--token` option:
```

      "voting_power": 100183904974426457056635,
      "consensus_key": "0xa6b9f7113a95aa40a26571f4e26f2a971910c6ff8b067dccee61e53108868f36afc0c79ed4f8dd972c5d0085181abfdb"
    },
    {
      "addr": "0x383A3c437d3F12f60E5fC990119468D3561EfBfc",
      "voting_power": 100183821357007074081774,
      "consensus_key": "0x89f62c1858723ed3d027d31026e37d6edfdf730e21a9315a1f4e54c50b363f341f88ee5b420606cce4ab124bfaa6c496"
    },
    {
      "addr": "0x9d28e40E9Ec4789f9A0D17e421F76D8D0868EA44",
      "voting_power": 100183707173309737349542,
      "consensus_key": "0xae6013d1ad0f8e1a1ed62c68ff3ef09461e32e95a7f5f4ded6b9fb4dc7f866a5984727e8214fc92e85906df4627625a2"
    },
    {
      "addr": "0x3fe573552E14a0FC11Da25E43Fef11e16a785068",
      "voting_power": 100183640771710575053864,
      "consensus_key": "0x96843bfb82f6b5860aaa93d59b81ac8e08e0ff9f5a76c25e5e5e3bb127ca06d0cffb19fabb4e25305c6be4ad18c7ac7b"
    },
    {
      "addr": "0x99E2B4B27BDe92b42D04B6CF302cF564D2C13b74",
      "voting_power": 100183205992233464410356,
      "consensus_key": "0xb2eed3ac8ec307e0862cdf7b8435049a98d8c83721a6066308ec57a1d9ec7f9c4e4725872dbfc96a83d102a77b430eb0"

>     }

Reviewer: There is no point to including more than a few validators in this output. It takes up a lot of space in the page and serves no purpose! I would suggest editing by hand to trim to 1-3 items or else making use of ellipses...

Author: Agree. Edited down in commit cc12151c0bc12a8728925f8dab97a84859acb52d.

1829
```

:::

## getEpochInfo

Returns the current epoch info of the chain.    

### Parameters

None.

### Response

Returns an `epochInfo` object consisting of:

| Field | Datatype | Description |
| --| --| --|
| `committee` | `CommitteeMember[]` | an array of `CommitteeMember` objects recording the consensus committee of the epoch |
| `previousEpochBlock` | `uint256` | The previous epoch block number |
| `epochBlock` | `uint256` | The epoch block number |
| `nextEpochBlock` | `uint256` | The next epoch block number |
| `delta` | `uint256` | the current value for delta (omission failure) |

For each committee member [`validator identifier`](/concepts/validator/#validator-identifier) address, [voting power](/glossary/#voting-power), and [consensus key](/concepts/validator/#p2p-node-keys-autonitykeys) is returned. See [`getCommittee()`](/reference/api/aut/#getcommittee) for the description of `CommitteeMember` object properties.

### Usage

::: {.panel-tabset}
## aut

``` {.aut}
aut protocol epoch-info [OPTIONS]
```

:::

### Example

::: {.panel-tabset}
## aut

``` {.aut}
aut protocol epoch-info
{

>   "committee": [

Reviewer: As per previous comment, there is no point including 60+ validators in this output as it takes up space for no benefit AFAIK.

Author: Agree. Edited down in commit caa0b402c48bf030176c42f1bbface1534207b3e.

aut protocol version -r $RPC_URL
1
```
:::

## isUnbondingReleased

Returns a boolean flag specifying if stake for an unbonding request has been released or not.

### Parameters

| Field | Datatype | Description |
| --| --| --|
| `_unbondingID ` | `uint256` | the unique identifier of the unbonding request |

### Response

Returns `true` if unbonding is released and `false` otherwise.

### Usage

::: {.panel-tabset}
## aut

``` {.aut}
aut contract call [OPTIONS] isUnbondingReleased [PARAMETERS]
```

:::

### Example

::: {.panel-tabset}
## aut

``` {.aut}
aut contract call --abi ../scripts/abi/v1.0.2-alpha/Autonity.abi  --address 0xBd770416a3345F91E4B34576cb804a576fa48EB1  isUnbondin

> gReleased 2

Reviewer: There is an erroneous newline here:

```suggestion
aut contract call --abi ../scripts/abi/v1.0.2-alpha/Autonity.abi  --address 0xBd770416a3345F91E4B34576cb804a576fa48EB1  isUnbondingReleased 2
```

:::

## totalSupply

Returns the amount of Newton stake token issued (ERC-20).

You can return the total supply for an ERC-20 token contract using the `aut token` command group, which is for interacting with ERC-20 token contracts. For example, of a Liquid Newton account.

::: {.callout-note title="What is the difference between the total and circulating supply of Newton?" collapse="true"}

> Total supply is the total amount of Newton that has been minted. Circulating supply is simply the total supply minus the sum of any minted NTN that is locked and released into circulation over time by unlocking according to a schedule.

Reviewer 2: ```suggestion
Total supply is the total amount of Newton that has been minted. Circulating supply is simply the total supply minus the sum of any minted NTN that is locked and released into circulation over time according to a schedule.
```
